knoten/shape.py

```diff
@@ -0,0 +1,58 @@
+from knoten import csm, utils
+import spiceypy as spice
+import numpy as np
+import csmapi
+
+class Ellipsoid:
+    """
+    A biaxial ellipsoid shape model.
+    """
+
+    def __init__(self, semi_major, semi_minor = None):
+        """
+        Create an ellipsoid shape model from a set of radii
+
+        Parameters
+        ----------
+        semi_major : float
+                     The equatorial semi-major radius of the ellipsoid.
+        semi_minor : float
+                     The polar semi-minor radius of the ellipsoid.
+        """
+        self.a = semi_major
+        self.b = semi_major
+        self.c = semi_major
+
+        if semi_minor is not None:
+            self.c = semi_minor
```

Comment:
Could this also take a third axis argument? There are smaller bodies that define a triaxial shape for the ellipsoid. I think something like `(self, semi_major, semi_minor=None, median=None)`. Where the semi_major corresponds with a, semi_minor corresponds with c, and median corresponds with c. Doing the same setting that you have for c and the semi_minor with b and the median.